src/components/BatchAllocations/hooks/useBatchAllocationLogs/useBatchAllocationLogs.js

```diff
@@ -0,0 +1,50 @@
+import { useQuery } from 'react-query';
+
+import {
+  useNamespace,
+  useOkapiKy,
+} from '@folio/stripes/core';
+import { LIMIT_MAX } from '@folio/stripes-acq-components';
+
+import { FUND_UPDATE_LOGS_API } from '../../../../common/const';
+
+const DEFAULT_LOGS = [];
+
+export const useBatchAllocationLogs = (options = {}) => {
+  const query = 'cql.allRecords=1 sortby metadata.createdDate/sort.descending';
+  const limit = LIMIT_MAX;
```

Comment:
I think you can place this code inside of `searchParams` instead of creating separate consts